string classification
float64 probability
int64 x
int64 y
int64 width
int64 height
float32 distance
